doc/ref_arch/kubernetes/chapters/appendix-a.md

```diff
@@ -1,18 +1,26 @@
 [<< Back](../../kubernetes)
-# Appendix A - Guidance for VNF-to-CNF Transition
+# Appendix A - Guidelines For VNF Vendors
```

Comment:
The heading in the doc is "Guidance" not "Guidelines", suggest changing to "Guidance"

Reply:
@CsatariGergely - still need this typo corrected, then I can approve 👍

Reply:
Done